### PR TITLE
CBO-642: Re/Allocation page review

### PR DESCRIPTION
**Pull request by cleargif**
#### What
- implement the autocomplete on the page
- upgrade the page to use the v2 stylesheet

#### Ticket

[CBO-642](https://dsdmoj.atlassian.net/browse/CBO-642)

#### Why
DAC report

### CODE REVIEW

File: app/assets/stylesheets/v2/_autocomplete.scss [hunk 1]
```diff
@@ -39,7 +39,8 @@
 
 @media (min-width: 641px) {
   .tt-menu {
-    width: 50%;
+    // possibly still required
```
Comment (colinbruce) [inline app/assets/stylesheets/v2/_autocomplete.scss]: If it's not required can it be deleted now?

File: app/views/case_workers/admin/allocations/_filter_tasks.html.haml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 .form-group#filter-tasks.dtFilter.dtFilterAGFS{:data => {scheme: 'agfs'}}
   %fieldset.inline
-    %label.form-label{:for => "filterAGFS"}
+    %label.form-label-bold{:for => "filterAGFS"}
       = t('.case_types')
-    %select#filterAGFS.form-control{:name => "tasks"}
+    %select#filterAGFS.form-control{:name => "tasks", :style => "width:100%"}
```
Comment (colinbruce) [inline app/views/case_workers/admin/allocations/_filter_tasks.html.haml:5]: Inline styles?! 🤔

**Review by colinbruce — APPROVED** [review]
Generally fine, there's a couple of observations, but no major blockers
👍

File: app/views/case_workers/admin/allocations/_filter_tasks.html.haml (hunk 2 above)
Comment (cleargif) [reply app/views/case_workers/admin/allocations/_filter_tasks.html.haml:5]: I want to go over these views again at some point. will raise a ticket.

Comment (cleargif) [reply app/views/case_workers/admin/allocations/_filter_tasks.html.haml:5]: CBO-657

**Pull request merged**